ui/src/app/machines/components/ActionFormWrapper/MarkBrokenForm/MarkBrokenForm.tsx

```diff
@@ -28,11 +28,13 @@ type Props = {
 export const MarkBrokenForm = ({ setSelectedAction }: Props): JSX.Element => {
   const dispatch = useDispatch();
   const activeMachine = useSelector(machineSelectors.active);
-  const machineErrors = useSelector(machineSelectors.errors);
-  const errors = Object.keys(machineErrors).length > 0 ? machineErrors : null;
-  const { machinesToAction, processingCount } = useMachineActionForm(
-    NodeActions.MARK_BROKEN
-  );
+  const {
+    errors: machineErrors,
+    machinesToAction,
+    processingCount,
+  } = useMachineActionForm(NodeActions.MARK_BROKEN);
+  const errors =
+    Object.keys(machineErrors || {}).length > 0 ? machineErrors : null;
```

Comment:
Do you think this step is necessary? `formatErrors` should already get run in `ActionForm.tsx`. Anyway it wasn't introduced in this PR but if you think it's an easy driveby fix go ahead

Reply:
Yes, quite right, I'll remove it.